addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -418,39 +426,66 @@
-                            <i t-att-class="'fa fa-circle text-400 %s' % ('d-none' if slide_completed and answer['is_correct'] else '')"/>
-                            <i class="fa fa-times-circle text-danger d-none"></i>
-                            <i t-att-class="'fa fa-check-circle text-success %s' % ('d-none' if not (slide_completed and answer['is_correct']) else '')"></i>
-                        </label>
-                        <span t-esc="answer['text_value']"/>
-                    </a>
-                </t>
+            <div class="ml-auto o_wslides_js_quiz_edit_del" t-if="slide.channel_id.can_upload and not slide_completed" >
+                <i class="o_wslides_js_quiz_edit_question fa fa-pencil-square-o p-1"></i>
+                <i class="o_wslides_js_quiz_delete_question fa fa-trash p-1"></i>
             </div>
         </div>
-        <div class="alert alert-danger o_wslides_js_lesson_quiz_error mt-2 d-none" role="alert">
-            <span></span>
-            <button type="button" class="close" data-dismiss="alert" aria-label="Close">
-                <span aria-hidden="true">&amp;times;</span>
-            </button>
+        <div class="list-group">
+            <t t-foreach="question['answer_ids']" t-as="answer">
+                <a t-att-data-answer-id="answer['id']" href="#"
+                    t-att-data-text="answer['text_value']" t-att-data-is-correct="answer['is_correct']"
+                    t-att-class="'o_wslides_quiz_answer list-group-item list-group-item-action d-flex align-items-center %s' % ('list-group-item-success' if slide_completed and answer['is_correct'] else '')">
+                    <label class="my-0 d-flex align-items-center justify-content-center mr-2">
+                        <input type="radio"
+                            t-att-name="question['id']"
+                            t-att-value="answer['id']"
+                            class="d-none"
+                            t-att-disabled="True if not slide.channel_id.is_member or slide_completed else ''"/>
+                        <i t-att-class="'fa fa-circle text-400 %s' % ('d-none' if slide_completed and answer['is_correct'] else '')"/>
+                        <i class="fa fa-times-circle text-danger d-none"></i>
+                        <i t-att-class="'fa fa-check-circle text-success %s' % ('d-none' if not (slide_completed and answer['is_correct']) else '')"></i>
+                    </label>
+                    <span t-esc="answer['text_value']"/>
+                </a>
+            </t>
         </div>
-        <div class="o_wslides_js_lesson_quiz_validation pt-3"/>
+    </div>
+</template>
+
+<template id="lesson_content_quiz_add_buttons" name="Lesson: Quiz Add Buttons template">
+    <div class="o_wslides_js_lesson_quiz_new_question row mt-3">
+        <a id="quiz" t-attf-class="o_wslides_js_quiz_add o_wslides_js_quiz_add_quiz btn btn-light border ml-3 #{'d-none ' if slide.question_ids else ''}" role="button">
```

Comment:
Do you need that id ?
It seems a little weak without any namespacing.

If not necessary, just remove it.

Reply:
I do. It's used as an anchor when clicking on the 'Add Quiz' button in the course list, useful when the course is quite long and you don't want to scroll down the whole page.

Reply:
Then please properly namespace it to avoid potential collisions.
If someone introduces a generic "quiz" mechanic on the website and uses the same id as you, it won't work anymore.

Replace by something like "o_wslides_quiz_add".

And also, can't you use a simple class for the purpose you're describing?
If it's the case, you could use the existing class "o_wslides_js_quiz_add_quiz" instead.
Not sure but worth checking.

Reply:
After some testing I don't really need the ID...
There is a method who check the presence of the anchor in the URL and which called the method to add a new question but since there is a focus on the question input the anchor isn't needed.

So I changed the #quiz anchor in the URL by ?quiz_quick_create "fake" parameter